codeJE/Compare.C

```diff
@@ -37,6 +38,8 @@ Int_t Compare(TString pathFileO2 = "AnalysisResults_O2.root", TString pathFileAl
   //  vecSpecVecSpec.push_back(std::make_tuple("events", vecHisEvents, 4, 2));
   if (options.Contains("jets"))
     vecSpecVecSpec.push_back(std::make_tuple("jets", vecHisJets, 3, 2));
+  if (options.Contains("Events"))
+    vecSpecVecSpec.push_back(std::make_tuple("Events", vecHisEvents, 1, 1));
```

Comment:
```suggestion
  if (options.Contains("Events"))
    vecSpecVecSpec.push_back(std::make_tuple("Events", vecHisEvents, 1, 1));
```
Please use the existing lines above which are currently commented out.
Also please add the missing spaces around "jets" in the line `Contains("jets")`.